changelog: Add 1.33.0 release notes (#4391)
bottlerocket-os/bottlerocket: CHANGELOG.md

# v1.33.0 (2025-02-12)

## Release Highlights
* Remove aws-k8s-1.24 variants (https://github.com/bottlerocket-os/bottlerocket/issues/4359)
* Add support for NVIDIA Multi-Instance GPU (MIG) (https://github.com/bottlerocket-os/bottlerocket/issues/4252)

> 

Reviewer: Can we change these to be numerical links in the same way that we do for pull requests?

Author: Should I do it this way `[#4359](https://github.com/bottlerocket-os/bottlerocket/issues/4359)`? That's how we did in the bottom but there were other ways too like just the plain link.

Author: ohh looks like those numbers are unique and there is no overlap between PR and issues. I will do it similar to PRs and add the links at the bottom.

## OS Changes
* Add support for NVIDIA Multi-Instance GPU (MIG) ([#4306])

> * Migrate the default host containers to setting generator struct ([#4324])

Reviewer: I suggest modifying this to focus on the outcome of the change

```suggestion
* Allow default host containers to be changed without a migration ([#4324])
```

# v1.33.0 (2025-02-12)

## Release Highlights
* Remove aws-k8s-1.24 variants ([#4359])
* Add support for NVIDIA Multi-Instance GPU (MIG) ([#4252])

## OS Changes
* Add support for NVIDIA Multi-Instance GPU (MIG) ([#4306])
* Allow default host containers to be changed without a migration ([#4324])
* Add default managed Bootstrap container ([#4349])
* Correct migrations for public control containers. ([#4388])

## Build Changes

> * Update bottlerocket-kernel-kit to 1.1.0 ([#4386])

Reviewer 2: We can point as well to:
https://github.com/bottlerocket-os/bottlerocket/pull/4383
